backend/wallet/storage/user.py

```diff
@@ -89,6 +89,7 @@ def update_user_password(user_id: int, password_hash: str, salt: str) -> None:
     if user is not None:
         user.password_hash = password_hash
         user.password_salt = salt
+        user.password_reset_token_expiration = datetime.now() - timedelta(minutes=30)
```

Comment:
Do you just want to expire the token? If this is the only purpose, then why not to set it to None? IMHO, t is misleading this way.